kv/dist_sender.go

```diff
@@ -402,7 +402,7 @@ func (ds *DistSender) sendRPC(desc *proto.RangeDescriptor,
 	}
 
 	// Copy and rearrange the replicas suitably, then return the desired order.
-	replicas := proto.ReplicaSlice(append([]proto.Replica(nil), desc.Replicas...))
+	replicas := append(proto.ReplicaSlice{}, desc.Replicas...)
```

Comment:
I don't know. In general, I use `proto.ReplicaSlice(nil)` when I want an empty slice.